app/views/handlers/list_repeating_block.py

```diff
@@ -0,0 +1,58 @@
+from functools import cached_property
+
+from flask import url_for
+
+from app.views.handlers.list_action import ListAction
+
+
+class ListRepeatingBlock(ListAction):
```

Comment:
I've just compared with the card and we are asked to add `New each block type inside  repeating_blocks will be of type ListRepeatingQuestion`. So the new block should be called `ListRepeatingQuestion`?

Reply:
i could have sworn I was following the pattern of calling list actions [Action]Block but they're all Question, will change thanks!

Reply:
Renamed to ListCollectorQuestion 👍